pagopa/pn-bff
# implemented new api for digital domicile - pn-12535

## Short description

With the digital domicile feature, we need to manage the new channelType SERCQ and the new tos and privacy acceptance.

## List of changes proposed in this pull request

- Added new channelType SERCQ_SEND
- The getConsentsV1 api now call the getConsents micro-service api
- Changed the request body for the acceptTosPrivacyV1 api. In this way we can manage the acceptance also for tos and privacy different from overall ones
- Update pn-commons and pn-model dependency

## How to test

Run the clean install command -> run the tests and try the api on postman -> check that everything is ok
